addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/profiles/EnOceanProfileTypes.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.enocean.internal.profiles;
+
+import static org.openhab.binding.enocean.internal.EnOceanBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.CoreItemFactory;
+import org.eclipse.smarthome.core.thing.DefaultSystemChannelTypeProvider;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeBuilder;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeUID;
+import org.eclipse.smarthome.core.thing.profiles.StateProfileType;
+import org.eclipse.smarthome.core.thing.profiles.TriggerProfileType;
+
+/**
+ * Class holds ProfileTypeUIDs and ProfileTypes of EnOcean profiles
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public class EnOceanProfileTypes {
+
+    public static final ProfileTypeUID RockerSwitchToPlayPause = new ProfileTypeUID(BINDING_ID,
+            "rockerswitch-to-play-pause");
+
+    public static final ProfileTypeUID RockerSwitchFromOnOff = new ProfileTypeUID(BINDING_ID,
```

Comment:
Do we actually still NEED the "rockerswitch-from..." profiles? I thought that the corresponding channel typ can already be used by the handler to determine the correct behavior. You aren't mentioning them in the README either.

Reply:
I removed all profiles now. Logic of "rockerswitch-from..." profiles moved into the ClassicDeviceHandler where it belongs. I have updated the readme accordingly.